src/client/src/mockData/mockVsCodeApi.ts

```diff
@@ -212,6 +212,138 @@ const mockVsCodeApi = () => ({
             "*"
           );
           break;
+        case EXTENSION_COMMANDS.GET_PAGES:
+          window.postMessage(
+            {
+              command: EXTENSION_COMMANDS.GET_PAGES,
+              payload: {
+                pages: [
+                  {
+                    templateId: "wts.Page.React.Blank",
+                    name: "Blank",
+                    defaultName: "Blank",
+                    description:
+                      "A blank page for you to build your web application from scratch.",
+                    richDescription:
+                      "This is the most basic page. A blank canvas to mold into whatever you wish. The blank page leaves pretty much everything up to you.",
+                    author: "Microsoft",
+                    version: "1.0.0",
+                    icon:
+                      "C:Users\t-nijindAppDataLocalCoreTSTemplatesLocalEnv\0.0.0.0WebPagesReact.Blank.template.configicon.xaml",
```

Comment:
Just noticed this, may want to change the icon to some dummy text, as the extension handles it. Happens in many places.

Reply:
We changed this to empty strings in the mockvscode api for the frameworks icon if u wanna do that @njindal239

Reply:
Makes sense! Thanks for that :)